packages/nexrender-core/src/tasks/render.js

```diff
@@ -246,7 +246,7 @@ Estimated date of change to the new behavior: 2023-06-01.\n`);
                 });
 
                 clearTimeout(timeoutID);
-                return reject(new Error(outputStr || 'aerender.exe failed to render the output into the file due to an unknown reason'));
+                return reject(new RenderProcessError(code, signal, outputStr || 'aerender.exe failed to render the output into the file due to an unknown reason'));
```

Comment:
aerender.exe is specific to Windows environnement. Nexrender run also on macOS 😉